benchmarks/nel/project.yml

```diff
@@ -36,96 +47,94 @@ workflows:
     - evaluate
 
 commands:
-  - name: setup
-    help: Install dependencies
+  - name: download_mewsli9
+    help: Download Mewsli-9 dataset.
     script:
-      - "python -m pip install -r requirements.txt"
-    deps:
-      - "requirements.txt"
+      - bash scripts/datasets/download_mewsli-9.sh
+    outputs:
+      - assets/mewsli_9
 
   - name: preprocess
-    help: Preprocess test datasets
+    help: Preprocess test datasets.
     script:
-      - "unzip -q -d assets/reddit assets/reddit.zip"
-      - "python ./scripts/clean_data.py reddit"
+      - "python ./scripts/clean_data.py ${vars.dataset}"
     deps:
-      - "assets/reddit.zip"
+      - "assets/${vars.dataset}"
     outputs:
-      - "assets/reddit"
+      - "assets/${vars.dataset}"
```

Comment:
We could introduce a new directory for the cleaned corpora. Didn't do that because I thought it'd be more elegant to modify the corpus in-place. Should we do a separate directory with a cleaned corpus copy?

Reply:
I'd personally like that better, but I guess it's a matter of taste, so I don't mind strongly either way.

Reply:
Added `raw` and `clean` directories in `/assets/${vars.dataset}`.